### ISSUE TITLE
Uninitialised http_code variable in ooniprobe-android=1.1.2

### ISSUE THREAD

**Issue by darkk** [TheTorProject/ooniprobe-android#93]
I've found something like a bug in MK / oonirpobe-android while digging TheTorProject/ooni-pipeline#62

Interesting part of the measurement is:
```
        "method": "GET",
        "response": {
          "body": "",
          "response_line": "",
          "code": 2520491460
        }
```

It looks like uninitialised variable to me. It may be already fixed for sure, just close the ticket if it's so.

IDs that can be used to lookup [the measurement](https://s3.amazonaws.com/ooni-public/sanitised/2017-03-21/20170320T151801Z-NL-AS202952-web_connectivity-20170320T151809Z_AS202952_nlgC69Tyar66FNQJhQDIreRRkYVYUSYzszVxSvb2dIeWogDiJx-0.2.0-probe.json):
```
  "software_name": "ooniprobe-android",
  "software_version": "1.1.2",
  "report_filename": "2017-03-21/20170320T151801Z-NL-AS202952-web_connectivity-20170320T151809Z_AS202952_nlgC69Tyar66FNQJhQDIreRRkYVYUSYzszVxSvb2dIeWogDiJx-0.2.0-probe.json",
  "input": "http://cartercenter.org",
  "report_id": "20170320T151809Z_AS202952_nlgC69Tyar66FNQJhQDIreRRkYVYUSYzszVxSvb2dIeWogDiJx",
  "measurement_start_time": "2017-03-20 15:19:03",
```

**Comment by bassosimone** [comment]
Yes, the bug you mention has been fixed in measurement-kit/measurement-kit#1174.

**Comment by bassosimone** [comment]
(Thanks for reporting it, and good catch!)
